src/fides/api/ctl/routes/system.py

```diff
@@ -51,54 +50,13 @@
     verify_oauth_client_for_system_from_request_body_cli,
 )
 
-system_router = APIRouter(tags=["System"], prefix=f"{V1_URL_PREFIX}/system")
-system_connections_router = APIRouter(
+SYSTEM_ROUTER = APIRouter(tags=["System"], prefix=f"{V1_URL_PREFIX}/system")
+SYSTEM_CONNECTIONS_ROUTER = APIRouter(
     tags=["System"], prefix=f"{V1_URL_PREFIX}{SYSTEM_CONNECTIONS}"
 )
 
 
-def get_system(db: Session, fides_key: str) -> System:
```

Comment:
basically every function that touched the database directly got moved to `database.system`

This file is now purely the endpoint definitions as they were, no logic changes